static/templates/single_message.handlebars

```diff
@@ -60,6 +61,7 @@
         </div>
         <div class="message_expander message_length_controller" title="{{t 'See the rest of this message' }}">{{t "[More...]" }}</div>
         <div class="message_condenser message_length_controller" title="{{t 'Make this message take up less space on the screen' }}">{{t "[Condense this message]" }}</div>
+        <div class="message_reactions {{msg/id}}">{{ partial "message_reactions" }}</div>
```

Comment:
Remove the ` {{msg/id}}` class -- you probably don't want the message ID as a class :)